aci/resource_aci_cloudprivatelinklabel.go

```diff
@@ -0,0 +1,191 @@
+package aci
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/ciscoecosystem/aci-go-client/v2/client"
+	"github.com/ciscoecosystem/aci-go-client/v2/models"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func resourceAciCloudPrivateLinkLabel() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAciCloudPrivateLinkLabelCreate,
+		UpdateContext: resourceAciCloudPrivateLinkLabelUpdate,
+		ReadContext:   resourceAciCloudPrivateLinkLabelRead,
+		DeleteContext: resourceAciCloudPrivateLinkLabelDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciCloudPrivateLinkLabelImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"cloud_service_epg_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+		})),
+	}
+}
```

Comment:
I don't think you can create a privatelink lable with just the name. you might need subnets as well.